src/pd/pd.rs

```diff
@@ -57,7 +57,8 @@ pub enum Task {
         pending_peers: Vec<metapb::Peer>,
         written_bytes: u64,
         written_keys: u64,
-        approximate_stat: Option<RegionApproximateStat>,
+        approximate_size: Option<u64>,
```

Comment:
Yes, since we separate the message `Msg::RegionApproximateSize` and `Msg::RegionApproximateRows`. And the message `Msg::RegionApproximateSize` will be sent in `SizeCheckObserver` while the the message `Msg::RegionApproximateRows` will be sent in `RowsCheckObserver`